### PR TITLE
change the item of the aws credentials for the openstack secrets

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform @alvaroaleman 
Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -1921,9 +1921,6 @@ secret_configs:
   - cluster: app.ci
     name: prometheus-auth-credentials
     namespace: prow-monitoring
-  - cluster: app.ci
```
Comment (alvaroaleman) [inline core-services/ci-secret-bootstrap/_config.yaml:1924]: Err I think this and everything below needs to stay, why did you remove it?

Comment (droslean) [reply core-services/ci-secret-bootstrap/_config.yaml:1924]: what the hell...

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 2]
```diff
@@ -2173,6 +2173,34 @@ secret_configs:
     name: registry-pull-credentials
     namespace: test-credentials
     type: kubernetes.io/dockerconfigjson
+  - cluster: app.ci
```
Comment (alvaroaleman) [inline core-services/ci-secret-bootstrap/_config.yaml:2176]: ?

**Comment by droslean** [comment]
@alvaroaleman updated. Git rebase sucks sometimes :dancers:

**Review by alvaroaleman — APPROVED** [review]
